WinCCOA_QualityChecks/scripts/libs/gedi/qualityCheck_ext.ctl

```diff
@@ -167,10 +167,7 @@ void tool_QualityGates_OpenResult()
 
 void open_Docu()
 {
-  if (getLocale(getActiveLang()) == "de_AT.iso88591" || getLocale(getActiveLang()) == "de_AT.utf8")
-    openUrl("https://www.winccoa.com/documentation/WinCCOA/3.18/en_US/index.html");
-  else
-    openUrl("https://www.winccoa.com/documentation/WinCCOA/3.18/en_US/index.html");
+  openUrl("https://github.com/siemens/CtrlppCheck/blob/main/docs/user.md");
```

Comment:
```suggestion
  openUrl("https://github.com/siemens/CtrlppCheck/blob/main/README.md");
```
Better use the readme, because it references user.md, but not vice-versa.

Reply:
I agree, but we shall rework the readme little bit in that case. But I think we have an other open issue to improve the documentation